docs/proposals/data-plane-config.md

```diff
@@ -0,0 +1,20 @@
+# Enhancement Proposal-929: Data Plane Dynamic Configuration
+
+- Issue: https://github.com/nginxinc/nginx-kubernetes-gateway/issues/929
+- Status: Provisional
+
+## Summary
+
+This proposal is intended to contain the design for how to dynamically configure the data plane for the NGINX Kubernetes
+Gateway project. Similar to control plane configuration, we should be able to leverage a custom resource definition to
+define data plane configuration.
+
+## Goals
+
+Define a CRD to dynamically configure various settings for the NKG data plane. The only initial option available will be
+log format.
```

Comment:
Log format implies nginx access log.
Log level implies nginx error log.

Reply:
let's use logging configuration as in ` The only initial option available will be
logging configuration`. It make sense to treat it as whole, so we arrive at a group of related fields, even though we implement only some of them for now.

as @brianehlert mentioned, it would make sense to keep in mind there are two logs in NGINX - access and error log.